components/Evaluator/ApplicantResponse.js

```diff
@@ -2,85 +2,100 @@ import React, { useState, useEffect } from 'react';
 import styled from 'styled-components';
 import { Title4 } from '../Typography';
 import ResponseInput from '../Assessment/responseInput';
-import { COLOR } from '../../constants';
+import { COLOR, ASSESSMENT_COLOR } from '../../constants';
 import { getResumeFile } from '../../utility/firebase';
 
 const Container = styled.div`
+  display: ${(p) => !p.shouldDisplay && 'none'};
```

Comment:
Nit: instead of having an empty value for this CSS property could we wrap the entire property? (ie. `${(p) => !p.shouldDisplay && 'display: none'};`

Not sure what best practices are though, but I've also seen this done so I'll leave this as a nit